File: Javascript/2 - mineur majeur.js
// Définition de la fonction verifierAge
function verifierAge() {
    // Demande à l'utilisateur d'entrer son âge via une boîte de dialogue
    // La saisie est convertie en nombre entier avec parseInt
    const age = parseInt(prompt("Entrez votre âge : "));

    // Vérifie si l'entrée n'est pas un nombre (NaN)
    if (isNaN(age)) {
        // Affiche un message d'alerte si l'utilisateur n'a pas entré un nombre valide
        alert("Veuillez entrer un nombre valide.");
    } else {
        // Utilise un opérateur ternaire pour vérifier si l'âge est supérieur ou égal à 18
        // Affiche "Majeur" si vrai, sinon "Mineur"
        alert(age >= 18 ? "Majeur" : "Mineur");
    }
}

// Appel immédiat de la fonction verifierAge pour exécuter le code
verifierAge();

File: python/2 - mineur majeur.py
# Votre script demande l’âge d’un utilisateur et vous affichez “majeur” ou “mineur”

# Demande à l'utilisateur d'entrer son âge
# La fonction input() récupère la saisie de l'utilisateur sous forme de chaîne
# int() convertit cette chaîne en un entier

age = int(input("Entrez votre âge : "))

# Vérifie si l'âge est supérieur ou égal à 18
if age >= 18:
        # Si la condition est vraie, affiche "Majeur"
        print("Majeur")
else:
        # Si la condition est fausse (c'est-à-dire que l'âge est inférieur à 18), affiche "Mineur"
        print("Mineur")
